File: webapp/src/app/models/dto/SessionRequestComment.ts
import { SessionRequest } from '..';

export interface SessionRequestComment {
  id: string;
  updatedDate: Date;
  isSuccessfulWorkout: boolean;
  givenSessionRequestId: string;
  receivedSessionRequestId: string;
  givenSessionRequest?: SessionRequest;
  receivedSessionRequest?: SessionRequest;
  comment: string;
}

File: server/Scheduler.Interfaces/Models/SessionRequestComment.cs
using System;

namespace Scheduler.Interfaces.Models
{
    public class SessionRequestComment
    {
        public string Id { get; set; }
        public DateTime UpdatedDate { get; set; }
        public bool IsSuccessfulWorkout { get; set; }
        public string GivenSessionRequestId { get; set; }
        public string ReceivedSessionRequestId { get; set; }
        public SessionRequest GivenSessionRequest { get; set; }
        public SessionRequest ReceivedSessionRequest { get; set; }
        public string Comment { get; set; }
    }


}
